slam-libraries/viam-cartographer/src/main.cc

```diff
@@ -17,7 +17,7 @@ void exit_loop_handler(int s) {
 
 int main(int argc, char** argv) {
     // glog only supports logging to files and stderr, not stdout.
-    FLAGS_alsologtostderr = 1;
+    FLAGS_logtostderr = 1;
```

Comment:
Changed this to log in stderr only - as opposed to both the log file and stderr. 

[q] Should we keep logging both to the log file and to stderr? Or should we keep this change?

Reply:
I haven't used the log files yet so unless you have I think we can remove

Reply:
I haven't either - I think I'd like to hear if @tessavitabile  has any objections to this change

Reply:
No objection